### PR TITLE
Implement new main page

### PR DESCRIPTION
**Pull request by torztomasz**
Resolves L2B-1975

### CODE REVIEW

File: packages/frontend/src/view/pages/home/common.ts [hunk 1]
```diff
@@ -13,14 +13,14 @@ export const L2_TRANSACTIONS_TABLE_PROPS = {
...
-  title: 'Latest forced transactions',
+  title: 'Forced transactions',
   entryShortNamePlural: 'transactions',
   entryLongNamePlural: 'forced transactions',
   path: '/forced-transactions',
 }
 
 export const OFFER_TABLE_PROPS = {
-  title: 'Available forced trade offers',
+  title: 'Available offers',
```
Comment (sz-piotr) [inline packages/frontend/src/view/pages/home/common.ts:23]: Naming offers "forced trade offers" was something specifically requested by SW long ago. I'd only make this change if they approved it this way.

File: packages/frontend/src/view/components/tables/l2-transactions/perpetual/PerpetualL2TransactionsTable.tsx [hunk 2]
```diff
@@ -20,16 +20,17 @@ import { PerpetualL2TransactionFreeForm } from './PerpetualL2TransactionFreeForm
...
   collateralAsset: CollateralAsset
+  showDetails: boolean
 }
 
 export function PerpetualL2TransactionsTable(
   props: PerpetualL2TransactionsTableProps
 ) {
   const columns: Column[] = [
-    { header: 'Age' },
+    { header: `ID` },
```
Comment (sz-piotr) [inline packages/frontend/src/view/components/tables/l2-transactions/perpetual/PerpetualL2TransactionsTable.tsx:30]: Maybe we could have an eslint rule for no unnecessary backticks? Not sure if a rule like this exists

File: packages/frontend/src/view/components/page/BreakpointIndicator.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+
+export function BreakpointIndicator() {
```
Comment (sz-piotr) [inline packages/frontend/src/view/components/page/BreakpointIndicator.tsx:3]: Not sure if this is the best place to put this component

File: packages/frontend/package.json [hunk 4]
```diff
@@ -59,6 +57,7 @@
     "postcss": "^8.4.14",
     "prettier": "^2.8.4",
     "prettier-plugin-tailwindcss": "^0.2.2",
-    "tailwindcss": "^3.0.24"
+    "@tailwindcss/container-queries": "^0.1.1",
```
Comment (sz-piotr) [inline packages/frontend/package.json:60]: Really cool that this is so widely supported https://caniuse.com/css-container-queries. Didn't even know this existed!

**Review by sz-piotr — APPROVED** [review]

File: packages/frontend/src/view/pages/home/common.ts (hunk 1 above)
Comment (torztomasz) [reply packages/frontend/src/view/pages/home/common.ts:23]: They've accepted it 👍

**Pull request merged**